Java/corecrypto/src/main/java/com/breadwallet/corecrypto/Wallet.java

```diff
@@ -89,11 +94,19 @@ private Wallet(BRCryptoWallet core, WalletManager walletManager, SystemCallbackC
     @Override
     public Optional<Transfer> createTransfer(com.breadwallet.crypto.Address target,
                                              com.breadwallet.crypto.Amount amount,
-                                             com.breadwallet.crypto.TransferFeeBasis estimatedFeeBasis) {
+                                             com.breadwallet.crypto.TransferFeeBasis estimatedFeeBasis,
+                                             Optional<Set<com.breadwallet.crypto.TransferAttribute>> attributes) {
```

Comment:
Did the IDE complain here? Optional isn't used as an argument (you instead annotate with @Nullable). That's just the convention in Java.

Reply:
Fixed.